game-frontend/src/components/help/index.tsx

```diff
@@ -16,7 +16,7 @@ const Help: FC = () => {
       <Dialog isOpen={isOpen} onDismiss={close} aria-label="help">
         <DialogWrapper>
           <Button className="close-button" onClick={close}>
-            <span>×</span>
+            <span>Close</span>
```

Comment:
Aren't we usually writing things in German in the app? :v: (See Help Dialog)

Reply:
oops, must have been looking at the 'save' button when i did that.